src/index.ts

```diff
@@ -57,7 +61,8 @@ export default function VitePluginInjectPreload(options: Options): Plugin {
                 : 'head-prepend'
             let href = attrs.href ? attrs.href : false
             if (href === false || typeof href === 'undefined') {
-              href = `/${asset}`
+              // prepend basePath to asset path, if href is not passed
+              href = `${basePath.replace(/\/$/, "")}/${asset}`
```

Comment:
👋 hello.
Sorry for jumping in like this. Just saw this change. Wouldn’t this replacement cause problems if the `basepath` is more than one folder deep? I would test this myself if I wouldn‘t be on the phone right now. I thought I‘ll write it here before I forget it. 😂

Reply:
I didn't take this change, I put directly the basePath like this :

```ts
href = `${basePath}${asset}
```

The `basePath` is clean and resolved by ViteJS, so it should be fine. Open an issue with your config if you have a problem with the 1.2 version with your base option :)

Reply:
Thanks, I missed that.
Sorry for bothering! Will check if I find any issues.